### PR TITLE
Update keyholder docs

### PR DESCRIPTION
**Pull request by jku**
This removes a lot of docs:
* EXISTING_SIGNER.md and NEW_SIGNER.md should be covered by tuf-on-ci/SIGNER.md
* Content from OVERVIEW.md was moved to tuf-on-ci/SIGNER.md

---

Happy to get feedback: I hope that any non-critical missing docs can be filed as new issues (in order to first get rid of inaccurate old docs).

### CODE REVIEW

File: playbooks/tuf-on-ci/SIGNER.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
-# Signer manual for Sigstore root-signing
+# Keyholder manual for Sigstore root-signing
 
-## One-time setup for new signers
+## Keyholder requirements
+
+1. Keyholders MUST subscribe to the [Sigstore Maintainer Calendar](https://calendar.google.com/calendar/u/0?cid=Y19ydjIxcDJuMzJsbmJoYW5uaXFwOXIzNTJtb0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) for upcoming root signing events.
```
Comment (Hayden-IO) [inline playbooks/tuf-on-ci/SIGNER.md]: Is this calendar still accurate? Do we want to deprecate it entirely and just rely on GitHub notifications from TUF-on-CI?

File: playbooks/tuf-on-ci/SIGNER.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+Root signing events are expected to occur about every 4-5 months. The next `v+1` version signing will be
+scheduled, and the `v+2` version will be tentatively scheduled.
+
+3. Obtaining a Yubikey for use as a signing key. If you need support to obtain one, please reach out to one of
+the maintainers or through Slack. Key configuration is described below.
+
+4. Participation in signing events! Keyholders are expected to participate in the scheduled root signing events.
+The steps will be announced in Slack and in the signing event PR. Typically keyholders are expected to review
+the signing event changes, sign and create a PR for their signature once per signing event. Additionally,
+keyholders should be "on-call" (available for quick pings during daytime hours) during the root signing window
```
Comment (Hayden-IO) [inline playbooks/tuf-on-ci/SIGNER.md]: Can we clarify the expectation here for how you'd be pinged? Via Slack?

File: playbooks/keyholders/OVERVIEW.md [hunk 3]
```diff
@@ -1,13 +0,0 @@
-# Keyholder Responsibilities
-
-This document outlines the responsibilities of a root keyholder.
-
-## Outline
-
-1. Keyholders MUST subscribe to the [Sigstore Maintainer Calendar](https://calendar.google.com/calendar/u/0?cid=Y19ydjIxcDJuMzJsbmJoYW5uaXFwOXIzNTJtb0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) for upcoming root signing events. Root signing events are expected to occur about every 4-5 months. The next `v+1` version signing will be scheduled, and the `v+2` version will be tentatively scheduled.
-
-2. A testing event will occur the week before the signing. Keyholders are required to communicate that they have completed the testing ([new](./NEW_SIGNER.md/#testing) and [existing](./EXISTING_SIGNER.md/#testing) testing) to the orchestrator through the [#sigstore-keyholder](https://sigstore.slack.com/archives/C03E4HP6RCK) Slack channel. All testing can occur asynchronously.
```
Comment (Hayden-IO) [inline playbooks/keyholders/OVERVIEW.md:9]: Are keyholders needed for test ceremonies?

File: playbooks/tuf-on-ci/SIGNER.md (hunk 1 above)
Comment (jku) [reply playbooks/tuf-on-ci/SIGNER.md]: I just copy-pasted, happy to remove if no-one maintains the calendar: The generated PRs and the repository status page could be a somewhat decent replacement.

File: playbooks/keyholders/OVERVIEW.md (hunk 3 above)
Comment (jku) [reply playbooks/keyholders/OVERVIEW.md:9]: I don't think that test ceremony is a concept we have use for anymore -- root-signing-staging should cover that

File: playbooks/tuf-on-ci/SIGNER.md (hunk 2 above)
Comment (jku) [reply playbooks/tuf-on-ci/SIGNER.md]: I will rewrite this: 
* I think the focus on quick pings is wrong -- as we can now take our time, we should not be in a rush anymore
* I added mention of unexpected signing events: that's definitely something we should be prepared for

File: playbooks/tuf-on-ci/SIGNER.md (hunk 1 above)
Comment (jku) [reply playbooks/tuf-on-ci/SIGNER.md]: asked for opinions on keyholders slack, will remove if no objections

Comment (jku) [reply playbooks/tuf-on-ci/SIGNER.md]: removing

**Review by Hayden-IO — APPROVED** [review]

**Pull request merged**